<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nymph</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="logWidget" native="true">
    <property name="geometry">
     <rect>
      <x>428</x>
      <y>12</y>
      <width>280</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QPlainTextEdit" name="logEdit">
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
       <property name="undoRedoEnabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmdEdit">
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertAtTop</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="mdiWidget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>351</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QMdiArea" name="mdiArea">
       <property name="background">
        <brush brushstyle="DiagCrossPattern">
         <color alpha="255">
          <red>191</red>
          <green>191</green>
          <blue>191</blue>
         </color>
        </brush>
       </property>
       <property name="viewMode">
        <enum>QMdiArea::TabbedView</enum>
       </property>
       <property name="documentMode">
        <bool>true</bool>
       </property>
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
       <property name="tabsMovable">
        <bool>true</bool>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Triangular</enum>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>745</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_new_file"/>
    <addaction name="action_open_file"/>
    <addaction name="separator"/>
    <addaction name="action_save_file"/>
    <addaction name="action_save_file_as"/>
    <addaction name="separator"/>
    <addaction name="action_close_file"/>
   </widget>
   <widget class="QMenu" name="menu_Run">
    <property name="title">
     <string>&amp;Run</string>
    </property>
    <addaction name="action_run"/>
    <addaction name="action_show_result"/>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_about_me"/>
    <addaction name="action_about_qt"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Run"/>
   <addaction name="menu_About"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_new_file"/>
   <addaction name="action_open_file"/>
   <addaction name="action_save_file"/>
   <addaction name="action_save_file_as"/>
   <addaction name="action_close_file"/>
   <addaction name="separator"/>
   <addaction name="action_run"/>
   <addaction name="action_show_result"/>
  </widget>
  <action name="action_new_file">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/Resources/icons/filenew.png</normaloff>:/Resources/icons/filenew.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New NL file</string>
   </property>
   <property name="toolTip">
    <string>Create an empty Nymph Lua file (.nl)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_save_file">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/Resources/icons/filesave.png</normaloff>:/Resources/icons/filesave.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save current editing NL file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_save_file_as">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/Resources/icons/filesaveas.png</normaloff>:/Resources/icons/filesaveas.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;as</string>
   </property>
   <property name="toolTip">
    <string>Save current editing NL file as new file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_open_file">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/Resources/icons/fileopen.png</normaloff>:/Resources/icons/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open NL file...</string>
   </property>
   <property name="toolTip">
    <string>Open an existing NL file...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_close_file">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/Resources/icons/window-close.png</normaloff>:/Resources/icons/window-close.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Close NL file</string>
   </property>
   <property name="toolTip">
    <string>Close current editing NL file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_run">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/Resources/icons/gtk-execute.png</normaloff>:/Resources/icons/gtk-execute.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Run</string>
   </property>
   <property name="toolTip">
    <string>Run current editing NL file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_show_result">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/Resources/icons/mail-send.png</normaloff>:/Resources/icons/mail-send.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Show result window</string>
   </property>
   <property name="toolTip">
    <string>Show Nymph viewer window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_about_me">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/Resources/icons/help-about.png</normaloff>:/Resources/icons/help-about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About me</string>
   </property>
   <property name="toolTip">
    <string>About the author and this software</string>
   </property>
  </action>
  <action name="action_about_qt">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/Resources/icons/qt.png</normaloff>:/Resources/icons/qt.png</iconset>
   </property>
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
